source/MaterialXCore/Definition.h

```diff
@@ -88,6 +88,25 @@ class NodeDef : public InterfaceElement
         return getAttribute(NODE_ATTRIBUTE);
     }
 
+    /// Return the element's output type.
+    const string& getType() const override
```

Comment:
Overrides getType() to return the type of the child output, if there is more than one, multioutput is returned.